src/Microsoft.Azure.NotificationHubs/WindowsNotification.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.Azure.NotificationHubs
     /// </summary>
     public sealed class WindowsNotification : Notification, INativeNotification
     {
+        const string contentType = "application/xml";
```

Comment:
The content-type for WNS is variable as noted in [our own documentation](https://docs.microsoft.com/en-us/rest/api/notificationhubs/send-wns-native-notification)

Set to `application/json;charset=utf-8` or `application/xml`. If the notification type (`X-WNS-Type`) is `wns/raw`, set to `application/octet-stream`.

Reply:
Currently it is set up [here ](https://github.com/Azure/azure-notificationhubs-dotnet/blob/d203f2b0c6cafb07b8f1e799c6983423f34c811e/src/Microsoft.Azure.NotificationHubs/WindowsNotification.cs#L134)